### ISSUE TITLE
Is clipboard meant not to work?

### ISSUE THREAD

**Issue by danielkrajnik** [foxlet/macOS-Simple-KVM#505]
Why would you disregard importance of clipboard, the very foundation of computing?

**Comment by Masamune3210** [comment]
One, no its not, computers were around well before the implementation of the clipboard perfectly fine
Two, you act like its some malicious attack on the feature instead of what it is, just a bug

**Comment by danielkrajnik** [comment]
Yes, I might have exagerrated it a little bit, but it would be really nice to see it mentioned somewhere whether clipboard sharing can be enabled, at all. Copy-pasting stuff is one of the most common activities (right after snapping images in Powerpoint).

**Comment by Masamune3210** [comment]
Well, jumping down someone's throat is not a good way to get them to sing.

**Comment by byemc** [comment]
> (Replying to @danielkrajnik)
Copy-pasting stuff is one of the most common activities (right after snapping images in Powerpoint).


But if you don't use PowerPoint? Also, are you trying to share a clipboard between macOS and Linux? Because the clipboard works perfectly fine in macOS 10.14 (Mojave)

By the way, for me the most common computer activities are playing games and programming - I haven't used PowerPoint for non-school work in years.

**Comment by juanbomfim22** [comment]
I'm not sure if it's what you were saying, but speaking of the use of <kbd>Ctrl</kbd>, in macOS it is replaced by the Super key on common keyboards (mine has the logo of Windows). So I struggled to copy-and-paste because I's doing <kbd>Ctrl + C</kbd> and <kbd>Ctrl + V</kbd> instead of <kbd>Super + C</kbd> and <kbd>Super + V</kbd>.

**Comment by byemc** [comment]
> I'm not sure if it's what you were saying, but speaking of the use of Ctrl, in macOS it is replaced by the Super key on common keyboards (mine has the logo of Windows). So I struggled to copy-and-paste because I's doing Ctrl + C and Ctrl + V instead of Super + C and Super + V.

You know what? you're probably right. Sometimes after using macOS I forget and try to use `Super + C` on Windows.
